packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -173,6 +174,7 @@ void main() {
       ),
     ]);
     expect(xcodeProjectInterpreter.version, isNull);
+    expect(xcodeProjectInterpreter.build, isNull);
```

Comment:
@jmagman  Do we want this? If the xcode version is invalid but the build version is not. Both need to be null?

Reply:
This check makes sense to me, checking that it's not parsing out something odd, or crashing.